Update User Guide - Add Attribute Table in read only mode (#9240)
geosolutions-it/MapStore2: docs/user-guide/layer-settings.md

In this page it is possible to:

* Change the **Title**

> * Set the translation of the layer title by opening the **Localize Text** popup through the <img src="../img/button/localize_button.jpg" class="ms-docbutton"/> button. In this way the language of the title changes according to the language that the user has set to [MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/)

Reviewer: ```suggestion
* Set the translation of the layer title by opening the **Localize Text** popup through the <img src="../img/button/localize_button.jpg" class="ms-docbutton"/> button. This way the language of the title changes according to the current language setting in [MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/)
```
